core/types/printing.cc

```diff
@@ -146,7 +154,10 @@ string AliasType::toStringWithTabs(const GlobalState &gs, int tabs) const {
 }
 
 string AliasType::show(const GlobalState &gs) const {
-    return fmt::format("<Alias: {} >", this->symbol.showFullName(gs));
+    // RBI generation: show() is supposed to print what the user typed, and the user doesn't
+    // type <Alias: SymbolName>!
+    return this->symbol.show(gs);
+    // return fmt::format("<Alias: {} >", this->symbol.showFullName(gs));
```

Comment:
Do you have an example of where this comes up? I'm having a tough time knowing what the right behavior is here absent an example.

Reply:
I believe this is covered by the rb-gen test. Go ahead and uncomment it and see if the test passes.